lib/execution_engine2/db/MongoUtil.py

```diff
@@ -263,6 +263,33 @@ def check_if_already_finished(job_status):
             return True
         return False
 
+    def update_jobs_to_queued(self, job_ids, scheduler_ids):
```

Comment:
Since there's a 1:1 mapping of job id to scheduler id maybe a dict or list of tuples would make more sense here? Then there's no way that 2 different size lists could be submitted

Reply:
Alright, we can do that, and then replace the check with a check to make sure that one of the job id and scheduler id combos doesn't have a None in it?

Reply:
Sounds good to me

Reply:
done

Reply:
:+1: